rad/chain.rad

```diff
@@ -126,8 +127,12 @@
             mod-state2))
     (list :ok mod-state3)))
 
-(def eval__ eval)
+(def eval__ "The eval in place when `chain.rad` is loaded." eval)
+
 (def eval
+  "An eval in which one can use `(:enter-chain url)` to make the
+  eval behave as that of a remote chain, and `:send` to send all
```

Comment:
I guess we'll leave switching all docs to this new format to a later PR?

Reply:
:100: https://github.com/oscoin/radicle/issues/181